setup.py

```diff
@@ -19,7 +19,7 @@
 
 name = "sqlalchemy-spanner"
 description = "SQLAlchemy dialect integrated into Cloud Spanner database"
-dependencies = ["sqlalchemy>=1.1.13, <=1.3.23", "google-cloud-spanner>=3.3.0"]
+dependencies = ["sqlalchemy>=1.1.13, <=1.3.23", "google-cloud-spanner>=3.3.0", "pytz"]
```

Comment:
Is pytz necessary here?

Reply:
Without it the tests could not pass. Removed.